### ISSUE TITLE
exporting images as png make them becomes corrupted

### ISSUE THREAD

**Issue by Charizardtheking** [mapeditor/tiled#3674]
Hi I am wondering if I could get some help as every time I try to export as an image png  the image becomes corrupted here is the file as an example 
![dfhjhsh](https://user-images.githubusercontent.com/131672784/234002465-6223757e-2d8c-4b62-a014-523697fa42a7.png)

**Comment by Charizardtheking** [comment]
The large transparent part is supposed to be like that as this tileset isn’t finished yet it’s not set on infinite map nor did I accidentally place anything outside the image I’m trying to export  as png to use rpg maker xp but that’s where the whole corrupted thing happens or saying that the file type isn’t supported

**Comment by Charizardtheking** [comment]
> Hmm, this image is incredibly big (64 x 567968 pixels), but it does open correctly for me in Tiled (on Linux, against Qt 6.4.3), with the following content at the top:
> 
> ![image](https://user-images.githubusercontent.com/531764/234005966-702d6d31-29d3-48a7-80d2-e495cf857ebb.png)
> 
> Followed by some more tiles and then a huge amount of blank space.
> 
> Since you say this image is from "Export As Image", could it be that you somehow have a layer with a huge offset in your map? This might explain the large transparent part of the image. Or is it an infinite map, where you accidentally placed some tiles very far away while being zoomed out?
> 
> I seems apart from the image height being so large, it is not actually corrupted.

The large transparent part is supposed to be like that as this tileset isn’t finished yet it’s not set on infinite map nor did I accidentally place anything outside the image I’m trying to export as png to use rpg maker xp but that’s where the whole corrupted thing happens or saying that the file type isn’t supported

**Comment by bjorn** [comment]
> The large transparent part is supposed to be like that as this tileset isn’t finished yet

Ok, I thought this image was the result of "Export As Image". When it is your tileset image, I think you went a little bit overboard with the empty area. Also, note that there is no reason at all to allocate a large transparent area at the bottom of your tileset, since you can always resize the image later as needed. Only changing the width of the tileset image is a potential issue, but changing the height does not invalidate any tile references.

And in that case, could you also attached the map you're using? I've set up a tileset with your huge tileset image, made a map and exported as image, but in my case that worked fine.

**Comment by Charizardtheking** [comment]
> > The large transparent part is supposed to be like that as this tileset isn’t finished yet
> 
> Ok, I thought this image was the result of "Export As Image". When it is your tileset image, I think you went a little bit overboard with the empty area. Also, note that there is no reason at all to allocate a large transparent area at the bottom of your tileset, since you can always resize the image later as needed. Only changing the width of the tileset image is a potential issue, but changing the height does not invalidate any tile references.
> 
> And in that case, could you also attach the map you're using (and if applicable, the tileset you've set up)? I've set up a tileset with your huge tileset image, made a map and exported as image, but in my case that worked fine.

Yeah the size is a lot as it’s still a work in process I just wanted to test it out and see if everything works on the program that I’m using  it’s probably best to finish the image and then worrying about exporting it later but I can try to attach the map let’s see if it’s just an issue with my system

**Comment by bjorn** [comment]
Please don't attach the map as csv file since that's not a format that can be loaded, as it is only an export format. Preferably use the tmx or json formats.

**Comment by bjorn** [comment]
@Charizardtheking I think GitHub supports uploading zip files, so best is to just archive all the files and upload them as such.

**Comment by Charizardtheking** [comment]
okay here is the file 
[ff.zip](https://github.com/mapeditor/tiled/files/11315435/ff.zip)

**Comment by bjorn** [comment]
@Charizardtheking Could you include all the files needed to reproduce the problem? Your map contains references to a dozen tilesets. Are these needed to reproduce the problem? If so, please include them in the zip file, both the `.tsx` files as well as their source images. But ideally, you could create a minimal map + tileset that produces a corrupted image for you when you try to export it as image.

To me it seems that the exported image, if it is the image you shared in your first message here, it simply very large and not corrupted. Could it be that the software you're trying to view or open that image in, just isn't able to render such large images?
